lua/treesitter-context.lua

```diff
@@ -315,6 +315,18 @@ local function get_gutter_width()
   return vim.fn.getwininfo(vim.api.nvim_get_current_win())[1].textoff
 end
 
+local window_scrolled_horizontal
+do
+  local leftcol
```

Comment:
Doesn't this need to be specific to a window?

Reply:
True, but since there is only 1 context window open for 1 relevant window at a time the variable can be reused. Now that i think of it there is 1 edge case where 2 windows are scrolled the same amount. Then the horizontal scroll will be missed after window change. I will fix that.